docs/input-variables.md

```diff
@@ -21,6 +21,7 @@ name                                | default                 | description
 etcdShape                           | VM.Standard1.1          | OCI shape for etcd nodes
 k8sMasterShape                      | VM.Standard1.1          | OCI shape for k8s master(s)
 k8sWorkerShape                      | VM.Standard1.2          | OCI shape for k8s worker(s)
+master_lb_enabled                   | true                    | enable/disable the k8s master load balancer. true: use the k8s master load balancer ip. false: use a nginx reverse proxy for k8s masters.
```

Comment:
Since `false` _still_ gets you an LB (nginx) , I wonder if this input variable should be renamed to something like `master_oci_lb_enabled` to be a bit more descriptive.

Reply:
done